python/delta/tests/test_pip_utils.py

```diff
@@ -73,10 +73,12 @@ def tearDown(self) -> None:
         shutil.rmtree(self.tempPath)
 
     def test_maven_jar_loaded(self) -> None:
-        packages: List[str] = self.spark.conf.get("spark.jars.packages").split(",")
-
+        packagesConf: Optional[str] = self.spark.conf.get("spark.jars.packages")
+        assert packagesConf is not None  # mypi needs this to assign type str
```

Comment:
```suggestion
        assert packagesConf is not None  # mypi needs this to assign type str from Optional[str]
```
does this help? since the types are with variable definition i don't know if docs mentioning the return type will help

Reply:
Maybe there is a better way to do this I had to google this